### DISCUSSION TITLE
Tech Stack of this project

### DISCUSSION THREAD

**Discussion by musfiq-dehan** [MusfiqDehan/company-2041#2] [Q&A] [answered]
Hey @MusfiqDehan ,

What tech stack you are using for making this project?

**Comment by MusfiqDehan** [comment, marked as answer]
Hey man, I am using two stacks in two separate folders for this project
1. HTML, CSS, CSS Media Query
2. HTML, CSS, and Bootstrap

Hopefully, you got your answer. Please let me know if you have any more questions. Thanks.
